core/src/main/java/com/netflix/conductor/core/events/DefaultEventProcessor.java

```diff
@@ -336,4 +313,26 @@ private Object getPayloadObject(String payload) {
         }
         return payloadObject;
     }
+
+    private RetryTemplate createRetryTemplate() {
```

Comment:
Should this move to `ConductorCoreConfiguration`?

Reply:
changed